src/Robo/Plugin/Commands/DeployCommands.php

```diff
@@ -73,5 +85,44 @@ public function deployDrupal(string $appDirPath, string $siteName = 'default', s
             // https://github.com/drush-ops/drush/issues/2449#issuecomment-708655673
             ->exec("$appDirPath/vendor/bin/drush config:import --yes")
             ->run();
+        // @todo Do we want/need this option if we can just check for environment variables?
+        // @todo This could be useful if a site wanted to opt out of this functionality.
```

Comment:
@adamzimmermann I'm a big fan of opting out of things! Could it be as simple as adding an additional key/value in the `$opts` array,  defaulting to `'optout' => false`?

Reply:
Isn't that what I'm doing with the `tugboat` option essentially? I'm confused. 🤔 

We could rename it to `slack` or something like that if we want.

Reply:
Reminder that not everyone can easily set their environment variables. So much depends on their infrastructure.

Reply:
@adamzimmermann I guess I misunderstood the comments or reacted to `rename this option then to be more generic` and thought about things that needn't be Tugboat-specific. Like a battery of tests I can magically skip with `optout => true`!